src/cache/cache_las.c

```diff
@@ -1149,8 +1149,8 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 			 * is higher or same as the las timestamp.
 			 */
 			WT_ASSERT(session,
-			    prepare_state != WT_PREPARE_LOCKED ||
-			    durable_timestamp != WT_TS_MAX ||
+			    prepare_state != WT_PREPARE_LOCKED &&
+			    durable_timestamp != WT_TS_MAX &&
```

Comment:
I'm just reading the comment too, so @bvpvamsikrishna should really be the reviewer.

Maybe write the last clause this way for a little better readability.
```
WT_ASSERT(session,
    prepare_state != WT_PREPARE_LOCKED &&
    durable_timestamp != WT_TS_MAX &&
    !(prepare_state == WT_PREPARE_INPROGRESS && durable_timestamp != 0));
```

Reply:
@keithbostic, I am not a fan of ```! (COND1 && COND2)```. I have updated the comment for better readability.